src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -171,32 +176,79 @@ def test_start_stop_job_status_loop(self):
 
         self.jc.start_job_status_loop()
         msg = self.jc._comm.last_message
-        self.assertIsNotNone(msg)
+        self.assertEqual(
+            {
+                "msg_type": "job_status_all",
+                "content": get_test_job_states(ACTIVE_JOBS)
+            },
+            msg["data"]
+        )
         self.assertTrue(self.jc._running_lookup_loop)
         self.assertIsNotNone(self.jc._lookup_timer)
 
         self.jc.stop_job_status_loop()
         self.assertFalse(self.jc._running_lookup_loop)
         self.assertIsNone(self.jc._lookup_timer)
 
+    @mock.patch(
+        "biokbase.narrative.jobs.jobcomm.jobmanager.clients.get", get_mock_client
+    )
+    def test_start_job_status_loop__cell_ids(self):
+        cell_2_jobs = get_cell_2_jobs(instance=False)
+        cell_ids = list(cell_2_jobs.keys())
+        for combo_len in range(1, len(cell_ids) + 1):
+            for combo in itertools.combinations(cell_ids, combo_len):
+                exp_job_ids = [
+                    job_id
+                    for cell_id, job_ids in cell_2_jobs.items()
+                    for job_id in job_ids
+                    if cell_id in combo and not JOBS_TERMINALITY[job_id]
+                ]
```

Comment:
Iterate through all cell combinations and all combination lengths and get the jobs expected to be in those cells

Reply:
Perhaps this should be a code comment

Reply:
good idea.